extensions/amp-ad-network-doubleclick-impl/amp-ad-network-doubleclick-impl-internal.md

```diff
@@ -47,7 +49,14 @@ Example - DoubleClick Ad
     data-slot="/6355419/Travel">
 </amp-ad>
 ```
-
+Example - With additional targeting
```

Comment:
How do we decide what examples should be here as opposed to feature specific pages?  I think we should keep these as simple as possible and instead users click into the feature otherwise it's possible they will go out of sync

Reply:
Removed all but the most generic example. Let me know what others you'd like to see here, and I'll add them.